src/codegen/doc/format.rs

```diff
@@ -401,12 +401,30 @@ pub(crate) fn find_method_or_function<
         env.analysis
             .find_record_by_function(env, search_record, search_fn)
     {
-        Some(gen_record_fn_doc_link(
+        Some(gen_type_fn_doc_link(
             record_info.type_id,
             fn_info,
             env,
             in_type,
         ))
+    } else if let Some((enum_info, fn_info)) =
+        env.analysis.find_enum_by_function(env, |_| true, search_fn)
```

Comment:
Realized these should probably also get an appropriate type search function. Something for tomorrow to check before merging.

Reply:
Ready now! Doesn't matter for normal docs where it's just a function name, but relevant for the new GI(?) docs where the type name is separately mentioned.